src/postman/operation.ts

```diff
@@ -33,12 +34,12 @@ export const transformPostmanCollectionOperation: PostmanCollectionHttpOperation
     path,
     summary: item.name,
     request: transformRequest(item.request),
-    responses: [],
+    responses: item.responses.map(transformResponse),
     /*
     servers: ...,
     security: ...,
     */
-  } as any;
```

Comment:
Ah, finally :)

Reply:
Aye, I could imagine how it annoys you (and me too).